tensorflow/python/keras/callbacks_test.py

```diff
@@ -23,6 +23,7 @@
 import json
 import os
 import re
+import requests
```

Comment:
This is already imported in a try block on line 60.

Reply:
removed